app/controllers/api/v2/bait_libraries_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # Provides a JSON API controller for BaitLibraryResource
+    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    class BaitLibrariesController < JSONAPI::ResourceController
+      # By default JSONAPI::ResourceController provides most the standard
+      # behaviour, and in many cases this file may be left empty.
+    end
+  end
+end
```

Comment:
Is this needed, adding bait library to the API? Bait library is set as part of the request isn't it (the thing we were working on together)? But maybe you are also separately reading / creating bait libraries, from Limber, independently of a request?

Reply:
OK I see you're accessing it in the Limber PR,  `app/models/labware_creators/pcr_cycles_binned_plate/csv_file/row.rb`

Could be set to immutable though potentially?

Reply:
Yeah I have to check the bait library name they put in the customer file matches a real bait library.
In terms of the controller it's the same as all the other v2 controllers.